person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+  # Getters and setters
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
- [ ] [OPTIONAL] Kindly put the attribute accessors as the first thing immediately after the class definition. Put these lines before the initialize method.